### PR TITLE
Update code HTML and style CSS for registration form

### PR DESCRIPTION
**Pull request by jinety**

### CODE REVIEW

File: practice-two/src/css/style.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
+form {
```
Comment (phuocduong-agilityio) [inline practice-two/src/css/style.css]: Do not style on element directly

File: practice-two/src/css/style.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  border: 5px solid black;
+  width: 712px;
+  height: 355px;
```
Comment (phuocduong-agilityio) [inline practice-two/src/css/style.css]: Do not set height

File: practice-two/src/css/style.css [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  margin: 0 auto;
+  text-align: right;
+}
+.container {
+  margin: 10px 20px;
+  text-align: left;
+}
+
+warn-msg {
```
Comment (phuocduong-agilityio) [inline practice-two/src/css/style.css]: This style won't work because it's not class

File: practice-two/src/css/style.css [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  width: 666px;
+  height: 20px;
+}
+
+#submitBtn {
+  margin: 0px 20px 10px;
+  padding: 10px 50px;
+  background-color: black;
+  color: white;
+  float: right;
```
Comment (phuocduong-agilityio) [inline practice-two/src/css/style.css]: Try another way to make button position to the right

File: practice-two/src/css/style.css [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+warn-msg {
+  color: red;
+  font-size: 15px;
+}
+
+.input-label {
+  display: block;
+  font-size: 20px;
```
Comment (phuocduong-agilityio) [inline practice-two/src/css/style.css]: All Color and Size should use variable

File: practice-two/src/css/style.css [hunk 6]
```diff
@@ -0,0 +1,35 @@
+form {
+  border: 5px solid black;
+  width: 712px;
+  height: 355px;
+  margin: 0 auto;
+  text-align: right;
+}
+.container {
+  margin: 10px 20px;
+  text-align: left;
```
Comment (phuocduong-agilityio) [inline practice-two/src/css/style.css]: Did we need this style? Default is left, as I remember

File: practice-two/src/index.html [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/style.css" />
+    <title>Information input form</title>
+  </head>
+  <body>
+    <form action="">
```
Comment (phuocduong-agilityio) [inline practice-two/src/index.html]: What is action for?

File: practice-two/src/index.html [hunk 8]
```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/style.css" />
```
Comment (phuocduong-agilityio) [inline practice-two/src/index.html:7]: Where is your reset file?

File: practice-two/.gitignore [hunk 9]
```diff
@@ -0,0 +1,4 @@
+.parcel-cache
+dist
+node_modules
+package-lock.json
```
Comment (hieungo-agilityio) [inline practice-two/.gitignore]: please check .editorconfig, does it work? This file is not added empty line at the end.

File: practice-two/package.json [hunk 10]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "practice-three",
```
Comment (hieungo-agilityio) [inline practice-two/package.json]: three?

File: practice-two/src/css/style.css [hunk 11]
```diff
@@ -0,0 +1,48 @@
...
+  --font-size-s: 18px;
+  --font-size-m: 24px;
+}
+
+.input-container-and-button {
+  border: 5px solid var(--primary-color-100);
+  width: 50%;
+  text-align: right;
+  margin: 0 auto;
+}
```
Comment (phuocduong-agilityio) [inline practice-two/src/css/style.css:18]: Missing Empty line

File: practice-two/src/css/style.css [hunk 12]
```diff
@@ -0,0 +1,48 @@
...
+  --primary-color-200: white;
+  --secondary-color-100: red;
+
+
+  /*FONT SIZE*/
+  --font-size-s: 18px;
+  --font-size-m: 24px;
+}
+
+.input-container-and-button {
```
Comment (phuocduong-agilityio) [inline practice-two/src/css/style.css]: Not good naming

File: practice-two/src/index.html [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+    <link rel="stylesheet" href="./css/reset.css">
```
Comment (phuocduong-agilityio) [inline practice-two/src/index.html]: Reset file should be before style

File: practice-two/src/index.html [hunk 14]
```diff
@@ -0,0 +1,32 @@
...
+    <title>Information input form</title>
+  </head>
+  <body>
+    <form>
+      <div class="input-container-and-button">
```
Comment (phuocduong-agilityio) [inline practice-two/src/index.html]: Do we need this div?

File: practice-two/src/css/reset.css [hunk 15]
```diff
@@ -0,0 +1,135 @@
...
+nav, 
+output, 
+ruby, 
+section, 
+summary,
+time, 
+mark, 
+audio, 
+video {
+	margin: 0;
```
Comment (hieungo-agilityio) [inline practice-two/src/css/reset.css]: check indentation.

**Pull request merged**